### DISCUSSION TITLE
BTV Cinema

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#477] [Show and tell]
### Channel ID

bTVCinema.bg

### Stream URL (optional)

https://cdn3.gledam.xyz/hls/btv-cinema/index.m3u8?e=1692278171&hash=ngCGE3dOiCgNKDT1XDXnVQ

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://cdn3.gledam.xyz/hls/btv-cinema/index.m3u8?e=1692278171&hash=ngCGE3dOiCgNKDT1XDXnVQ (offline)
